### ISSUE TITLE
replicating experiments

### ISSUE THREAD

**Issue by dribnet** [jacobandreas/codes#1]
Hi - I was hoping this repo would be suitable for replicating some of the paper's experiments, but it appears the code here is currently incomplete -  the data loading portions including `data.abstract.corpus` are dependencies, but are not currently included. Is this correct, or is there something I'm missing?

**Comment by jacobandreas** [comment]
No, I just forgot to check in all the data and task loading code. I'm cleaning these up and writing instructions for running the experiments---should have this all ready soon.

**Comment by jacobandreas** [comment]
OK, should be good to go for colors and driving---if you need birds I can send you the file directly. I'm still trying to pin down a couple of nondeterminisims in the training code, so numbers may not be exactly the same. Qualitative behavior is stable.

**Comment by dribnet** [comment]
Thanks, confirmed.  I'm now able to run the color_train config. I had to make a few changes to get it run in python3, which I've offered in #2.

Here's some notes on my process that could help others:

 * installed pyyaml dependency
 * commented out import cairo in color.py (only needed for visualize)
 * follow [README instructions](https://github.com/jacobandreas/neuralese#colors) for getting munroecorpus and also `mkdir experiments`
 * `./run.sh configs/color_train.yaml` (perhaps after tweaking run.sh)
 * if there is an `indent error` in munroecorpus, just fix the one offending line - replacing spaces with a tab
 * results/log in `experiments/color_train`. be sure to check run.log for errors
